pages/03.Howtos/11.dns-zone-transfer/docs.en.md

```diff
@@ -0,0 +1,149 @@
+| links                    | {u'self': u'https://api.infra.sys11cloud.net:9001/v2/zones/tasks/transfer_accepts/7ade2756-f726-48a6-a1e6-57463595ff2b', u'zone': u'https://api.infra.sys11cloud.net:9001/v2/zones/db1228fc-d955-4cc2-a6f8-23ad91024b20'} |
+| project_id               | fedcba9876543210fedcba9876543210                                                                                                                                                                                          |
+| status                   | COMPLETE                                                                                                                                                                                                                  |
+| updated_at               | 2019-06-28T16:01:28.000000                                                                                                                                                                                                |
+| zone_id                  | db1228fc-d955-4cc2-a6f8-23ad91024b20                                                                                                                                                                                      |
+| zone_transfer_request_id | ca6b190e-376b-4075-9dd6-a14d451212da                                                                                                                                                                                      |
++--------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+If everything was correct, the zone transfer is now complete and the zone has appeared in the recipients zone list and disappeared from the donors zone list.
+
+
+
+### Conclusion
+
+We have transferred a zone from one project to another.
+
+```shell
+$ openstack zone list
++--------------------------------------+--------------------------+---------+------------+--------+--------+
+| id                                   | name                     | type    |     serial | status | action |
++--------------------------------------+--------------------------+---------+------------+--------+--------+
+| db1228fc-d955-4cc2-a6f8-23ad91024b20 | transfer.domain.example. | PRIMARY | 1561735620 | ACTIVE | NONE   |
++--------------------------------------+--------------------------+---------+------------+--------+--------+
+```
+
+This can be useful to resolve a dispute peacefully, but it makes sense over deleting and recreating the zone especially if the zones recordsets shall be retained without interrupt or the zones are subzones that cannot otherwise be created in another project.
```

Comment:
```suggestion
This can be useful to resolve a dispute peacefully, but it makes sense over deleting and recreating the zone especially if the zones recordsets shall be retained without interruption or the zones are subzones that cannot otherwise be created in another project.
```